extractor/src/extract/extractor.py

```diff
@@ -13,17 +13,17 @@
 from arkhn_monitoring import Timer
 
 
-SQL_RELATIONS_TO_METHOD = {
-    "<": "__lt__",
-    "<=": "__le__",
-    "<>": "__ne__",
-    "=": "__eq__",
-    ">": "__gt__",
-    ">=": "__ge__",
+SQL_RELATIONS_TO_METHOD: Dict[str, Callable[[AlchemyColumn, str], Callable]] = {
+    "<": lambda col, value: col.__lt__(value),
+    "<=": lambda col, value: col.__le__(value),
+    "<>": lambda col, value: col.__ne__(value),
+    "=": lambda col, value: col.__eq__(value),
+    ">": lambda col, value: col.__gt__(value),
+    ">=": lambda col, value: col.__ge__(value),
     # not handled yet
     # "BETWEEN": "",
-    "IN": "in_",
-    "LIKE": "like",
+    "IN": lambda col, value: col.in_(value.split(",")),
```

Comment:
The input from pyrog must be `key1,key2` and not `(key1,key2)`, correct ? We might want to add a warning or a check for pyrog users

Reply:
Yes you are right I am going to add a usage hint on pyrog front